tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/SupplierSectionSteps.cs

```diff
@@ -86,16 +90,75 @@ public async Task ThenTheResponseContainsTheFollowingSupplierAddress(Table table
                 Line3 = address.Value<string>("line3"),
                 Line4 = address.Value<string>("line4"),
                 Line5 = address.Value<string>("line5"),
-                Town =  address.Value<string>("town"),
-                County =  address.Value<string>("county"),
-                Postcode =  address.Value<string>("postcode"),
-                Country =  address.Value<string>("country")
+                Town = address.Value<string>("town"),
+                County = address.Value<string>("county"),
+                Postcode = address.Value<string>("postcode"),
+                Country = address.Value<string>("country")
             };
 
             var expected = table.CreateInstance<SupplierAddressTable>();
             actual.Should().BeEquivalentTo(expected);
         }
 
+        [When(@"the user makes a request to update the supplier with order ID (.*)")]
+        public async Task WhenTheUserMakesARequestToUpdateTheSupplierWithOrderId(string orderId, Table table)
+        {
+            var supplierTable = table.CreateInstance<SupplierSectionTable>();
+            var addressTable = _context["SupplierAddress"];

```

Comment:
We should check if these keys exist, and throw a nice helpful error if they don't, so the person using the step knows what went wrong